src/shared-constants.h

```diff
@@ -65,6 +65,8 @@ extern Name EXIT;
 extern Name SHARED;
 extern Name EVENT;
 extern Name ATTR;
+extern Name WASI;
+extern Name WASI_UNSTABLE;
```

Comment:
These probably don't belong in a header unless they are at least used by multiple source files.

Reply:
I think one of my next PRs will use them, as I add more wasi support. If it turns out no then I can remove these.